randovania/cli/database.py

```diff
@@ -438,6 +439,54 @@ def trick_usage_documentation_command(sub_parsers: _SubParsersAction) -> None:
     parser.set_defaults(func=trick_usage_documentation_logic)
 
 
+def features_per_node_command_logic(args: Namespace) -> None:
+    """Reports features for given pickup nodes, taking into account area and node specific features"""
+    db = load_game_description(args)
+    pickup_node_count = 0
+    featureless_nodes = []
+    no_features = 0
+    one_features = 0
+    two_features = 0
+    threeplus_features = 0
+    for region in db.region_list.regions:
```

Comment:
since you never use this, you could also just do `for area ion db.region_list.all_areas:`